web/components/inputs/FcSearchBarRequests.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-text-field
+    v-model="internalValue.query"
+    append-icon="mdi-magnify"
+    class="fc-search-bar-requests flex-grow-0 flex-shrink-0"
+    dense
+    hide-details
+    label="Search"
+    outlined>
+    <template v-slot:prepend>
+      <v-select
+        v-model="internalValue.column"
+        class="fc-search-bar-requests-column font-weight-regular mt-0 pt-0 title"
+        dense
+        hide-details
+        :items="itemsColumn"
+        outlined>
+      </v-select>
+    </template>
+  </v-text-field>
+</template>
+
+<script>
+import FcMixinVModelProxy from '@/web/mixins/FcMixinVModelProxy';
+
+export default {
+  name: 'FcSearchBarRequests',
+  mixins: [FcMixinVModelProxy(Object)],
+  props: {
+    columns: Array,
+    searchKeys: Object,
```

Comment:
this setup makes this viable as a generic search bar control for `FcDataTable`, as we're just reusing the same `columns` and `searchKeys`.

(that said: we should definitely not refactor it until we come across a second use case.)